vaapi/vaapidisplay_unittest.cpp

```diff
@@ -154,6 +164,12 @@ VAAPI_DISPLAY_TEST(CreateInvalidHandleVA) {
 
 #if defined(__ENABLE_X11__)
 VAAPI_DISPLAY_TEST(CreateX11) {
+
+    if (!haveX11Display()) {
+        //we do not need test x11 related tings if we do not have x11 display
```

Comment:
Can we log the test as "skipped" in this case?  See this [intel-vaapi-driver test](https://github.com/01org/intel-vaapi-driver/blob/c4f333bd658b5752bb6afc7935dc65644201dfd8/test/i965_jpeg_decode_test.cpp#L180) for an example...  the [RecordProperty](https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#logging-additional-information) method is part of GTest.  Also, see [getFullTestname](https://github.com/01org/intel-vaapi-driver/blob/c4f333bd658b5752bb6afc7935dc65644201dfd8/test/i965_test_fixture.cpp#L27) method.